common/environment/setup/sourcepkg.sh

```diff
@@ -8,6 +8,7 @@ unset -v configure_script configure_args wrksrc build_wrksrc create_wrksrc
 unset -v make_build_args make_check_args make_install_args
 unset -v make_build_target make_check_target make_install_target
 unset -v make_cmd meson_cmd gem_cmd fetch_cmd make_check_pre
+unset -v CMAKE_GENERATOR
```

Comment:
Put this under the `CC` block, as it's also an env var we export to children processes instead of only used by xbps-src.

Reply:
Hope I understood this correctly :)

Reply:
You did :)